backend/dal/Migrations/v01.08.01/Up/PreUp/01-Buildings.sql

```diff
@@ -0,0 +1,423 @@
+FROM dbo.[Buildings] p
+JOIN dbo.[Addresses] a ON p.AddressId = a.Id
+WHERE a.[AdministrativeArea] IN ('Coast Mountains', 'Terrace', 'Stikine', 'Stewart', 'Alice Arm', 'Cranberry Junction', 'Gitanyow', 'Kispiox', 'Hazelton', 'New Hazelton', 'Rosswood', 'Lakelse Lake', 'Kitimat', 'Nechako', 'Kitamaat Village', 'Kildala Arm', 'Kemano', 'Kemano Beach', 'Butedale', 'Klemtu')
+    AND p.[AgencyId] = @CMBId
+
+UPDATE p
+    SET p.[AgencyId] = (SELECT [Id] FROM dbo.[Agencies] WHERE [Code] = 'SD 83')
+FROM dbo.[Buildings] p
+JOIN dbo.[Addresses] a ON p.AddressId = a.Id
+WHERE a.[AdministrativeArea] IN ('North Okanagan-Shuswap', 'Albas', 'St. Ives', 'Saint Ives', 'Sicamous', 'Tappen', 'Canoe', 'Grindrod', 'Enderby', 'Salmon Arm', 'Spallumcheen', 'Armstrong', 'Falkland', 'Scotch Creek')
+    AND p.[AgencyId] = @CMBId
+
+UPDATE p
+    SET p.[AgencyId] = (SELECT [Id] FROM dbo.[Agencies] WHERE [Code] = 'SD 84')
+FROM dbo.[Buildings] p
+JOIN dbo.[Addresses] a ON p.AddressId = a.Id
+WHERE a.[AdministrativeArea] IN ('Vancouver Island West', 'Gold River', 'Tahsis', 'Zeballos', 'Kyuquot', 'Esperanza')
+    AND p.[AgencyId] = @CMBId
+
+UPDATE p
+    SET p.[AgencyId] = (SELECT [Id] FROM dbo.[Agencies] WHERE [Code] = 'SD 85')
+FROM dbo.[Buildings] p
+JOIN dbo.[Addresses] a ON p.AddressId = a.Id
+WHERE a.[AdministrativeArea] IN ('Vancouver Island North', 'Holberg', 'Quatsino', 'Coal Harbour', 'Port Hardy', 'Sointula', 'Alert Bay', 'Port McNeill', 'Port Alice', 'Telegraph Cove', 'Nimpkish', 'Woss', 'Kingcome Inlet', 'Minstrel Island', 'Sullivan Bay')
+    AND p.[AgencyId] = @CMBId
+
+UPDATE p
+    SET p.[AgencyId] = (SELECT [Id] FROM dbo.[Agencies] WHERE [Code] = 'SD 87')
+FROM dbo.[Buildings] p
+JOIN dbo.[Addresses] a ON p.AddressId = a.Id
+WHERE a.[AdministrativeArea] IN ('Stikine', 'Dease Lake', 'Atlin', 'Cassiar', 'Tulsequash', 'Telegraph Creek', 'Boulder City', 'Lower Post', 'Kinaskan Lake', 'Cry Lake')
+    AND p.[AgencyId] = @CMBId
+
+UPDATE p
+    SET p.[AgencyId] = (SELECT [Id] FROM dbo.[Agencies] WHERE [Code] = 'SD 91')
+FROM dbo.[Buildings] p
+JOIN dbo.[Addresses] a ON p.AddressId = a.Id
+WHERE a.[AdministrativeArea] IN ('Francois Lake', 'Nechako Lakes', 'Bear Lake', 'Tchentlo Lakes', 'Takla Lake', 'Fort Babine', 'Smithers Landing', 'Granisle', 'Topley Landing', 'Babine Lake', 'Topley', 'Chuchi Lake', 'Trembleur Lake', 'Grand Rapids', 'Tachie', 'Pinchi', 'Fort St. James', 'Dog Creek', 'Donald Landing', 'Decker Lake', 'Bulkley Lake', 'Burns Lake', 'Fraser Lake', 'Fort Fraser', 'Vanderhoof', 'Fancois Lake', 'Grassy Plains', 'Tahtsa Lake', 'Whitesail Reach', 'Ootsa Lake', 'Whitesail Lake', 'Eutsuk Lake', 'Cheslatta Lake', 'Newstubb Lake', 'Natalkuz Lake')
+    AND p.[AgencyId] = @CMBId
+
+UPDATE p
+    SET p.[AgencyId] = (SELECT [Id] FROM dbo.[Agencies] WHERE [Code] = 'SD 92')
+FROM dbo.[Buildings] p
+JOIN dbo.[Addresses] a ON p.AddressId = a.Id
+WHERE a.[AdministrativeArea] IN ('Nisga''a', 'Kitwanga', 'Nass Camp', 'New Aiyansh', 'Laxgalts''ap')
```

Comment:
'Nisga''a' and 'Laxgalts''ap' may cause issues

Reply:
at least in gitub they appear to use the same chars.

Reply:
The double single quotes are an escape and don't cause any known issues.